packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -84,23 +68,6 @@ export function sequenceFieldToDelta<TNodeChange>(
                     out.pushContent(deleteMark);
                     break;
                 }
```

Comment:
The removal the "Delete" code needs to process the child changes because there may be some "MoveOut" marks in it.
I'm okay with not worring about it in this PR, but I thought I'd bring it up since that may have been an oversight.
Same for revive.

Reply:
Let's do this in a separate PR. I think we already have a task for fixing issues with moves in deltas, and it might make sense to add tests for scenarios like this then.